src/recipes/heartbeat-by-user-first-impression/index.js

```diff
@@ -245,23 +247,22 @@ let run = function (state, extras) {
     }
   };
 
-  let breaks = [1.0];
-
 
+  let channel_breaks = {'aurora':[.3]};
```

Comment:
why aren't you just taking everyone from aurora?  There is no other live survey.